src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static final String SEPARATE = System.lineSeparator();
 
+    public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Please, split logic of getStatistic method to 3 separate methods ([common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#divide-program-logic))

readFile() - returns content
generateReport() -(you pass contents here)
writeToFile()-(you pass report and filename here )